network-config-analyzer/ConnectionSet.py

```diff
@@ -409,6 +416,19 @@ def add_all_connections(self, excluded_protocols=None):
             else:
                 self.allowed_protocols[protocol] = True
 
+    def is_all_connections(self):
```

Comment:
I suggest a function `check_if_all_connections()` that will also take care of setting the `allow_all` flag and the `allowed_protocol` map. This will prevent duplicating code.

Reply:
done